Gems/PhysX/Code/Source/System/PhysXSystem.cpp

```diff
@@ -41,6 +45,51 @@ namespace PhysX
     AZ_CVAR(bool, physx_reportTimestepWarnings, false, nullptr, AZ::ConsoleFunctorFlags::Null, "A flag providing ability to turn on/off reporting of PhysX timestep warnings");
 #endif
 
+    // A helper function.
+    AZ::Debug::PerformanceCollector::DataLogType GetDataLogTypeFromCVar(const AZ::CVarFixedString& newCaptureType)
+    {
+        if (newCaptureType[0] == 'a' || newCaptureType[0] == 'A')
+        {
+            return AZ::Debug::PerformanceCollector::DataLogType::LogAllSamples;
+        }
+        else
+        {
+            return AZ::Debug::PerformanceCollector::DataLogType::LogStatistics;
+        }
+    }
+
+    AZ_CVAR(AZ::u32, physx_metricsFrameCountPerCaptureBatch, 60,
+        [](const AZ::u32& newValue)
+        {
+            PhysX::GetPhysXSystem()->GetPerformanceCollector()->UpdateFrameCountPerCaptureBatch(newValue);
+        },
+        AZ::ConsoleFunctorFlags::DontReplicate, "Number of frames in which performance will be measured per batch.");
+
+    AZ_CVAR(AZ::u32, physx_metricsNumberOfCaptureBatches,
+        0, // Starts at 0, which means "do not capture performance data". When this variable changes to >0 we'll start performance capture.
+        [](const AZ::u32& newValue)
+        {
+            PhysX::GetPhysXSystem()->GetPerformanceCollector()->UpdateNumberOfCaptureBatches(newValue);
+        },
+        AZ::ConsoleFunctorFlags::DontReplicate, "Collects and reports PhysX performance in this number of batches.");
+
+    AZ_CVAR(AZ::CVarFixedString, physx_metricsDataLogType,
+        "statistical", // (s)(S)tatistical Summary (average, min, max, stdev) / (a)(A)ll Samples. Default=s.
+        [](const AZ::CVarFixedString& newValue)
+        {
+            PhysX::GetPhysXSystem()->GetPerformanceCollector()->UpdateDataLogType(GetDataLogTypeFromCVar(newValue));
+        },
+        AZ::ConsoleFunctorFlags::DontReplicate, "Defines the kind of data collection and logging. "
+        "If starts with 's' it will log statistical summaries, if starts with 'a' will log each sample of data (high verbosity).");
```

Comment:
suggestion: Is it worth mentioning the `s`/`S` bit from the code comment in this block for users?

Reply:
Yeah, merged the code comment with the description. No need to duplicate the same info. 
Thanks!

Reply:
Nice one thanks!